foxy/distribution.yaml

```diff
@@ -5710,6 +5710,21 @@ repositories:
       url: https://github.com/uos/rospy_message_converter.git
       version: foxy
     status: maintained
+  rosros:
```

Comment:
I really don't think we should have a project called `rosros` in the official distribution.  It is far too generic, and unclear on what it does.  Please consider renaming this to be more descriptive of what it does.  See https://www.ros.org/reps/rep-0144.html for the rules about naming packages.

Reply:
Understandable.. I assume the package name in the ROS index does not necessarily need to match the package name in its source repository?

Would `python3-rosros` be acceptable for the ROS index?

Reply:
No, that does not resolve any of the issues with the name. Please read the naming guidelines linked above. The name should be at least unique and preferably informative about the contents. Repeating the name of the larger project name twice(which is explicitly said not to use in your package name "A package name should not contain ros as it is redundant.") as a name does not help anyone know what the package is or what it does.